src/main/java/com/hedera/exchange/HederaNetworkCommunicator.java

```diff
@@ -88,14 +103,13 @@
      * @param client
      * @param ertParams
      * @return Latest AddressBook from the Hedera Network
-     * @throws HederaStatusException
      * @throws TimeoutException
      * @throws InterruptedException
      */
-    public static ERTAddressBook updateExchangeRateFile(final ExchangeRate exchangeRate,
+    public ERTAddressBook updateExchangeRateFile(final ExchangeRate exchangeRate,
                                                         final ExchangeRate midnightExchangeRate,
                                                         Client client,
-                                                        ERTParams ertParams) throws HederaStatusException, TimeoutException, InterruptedException {
+                                                        ERTParams ertParams) throws Exception {
```

Comment:
why did you change it to `Exception` instead of leaving the specialized exceptions?